app/uk/gov/hmrc/cgtpropertydisposals/service/TaxEnrolmentService.scala

```diff
@@ -57,41 +68,76 @@ class TaxEnrolmentServiceImpl @Inject()(
         case Right(httpResponse) => httpResponse
       }
 
+  def makeES6call(
+    updateVerifiersRequest: UpdateVerifiersRequest
+  )(implicit hc: HeaderCarrier): Future[HttpResponse] =
+    taxEnrolmentConnector
+      .updateVerifiers(updateVerifiersRequest)
+      .value
+      .map {
+        case Left(error)         => HttpResponse(999, Some(JsString(error.toString)))
+        case Right(httpResponse) => httpResponse
+      }
+
   override def allocateEnrolmentToGroup(taxEnrolmentRequest: TaxEnrolmentRequest)(
     implicit hc: HeaderCarrier
   ): EitherT[Future, Error, Unit] =
     for {
-      httpResponse <- EitherT.liftF(makeTaxEnrolmentCall(taxEnrolmentRequest))
-      result <- EitherT.fromEither(handleTaxEnrolmentResponse(httpResponse)).leftFlatMap[Unit, Error] { error: Error =>
-                 logger
-                   .warn(
-                     s"Failed to allocate enrolments due to error: $error; will store enrolment details"
-                   )
-                 taxEnrolmentRepository
-                   .insert(taxEnrolmentRequest)
-                   .leftMap(error => Error(s"Could not store enrolment details: $error"))
+      httpResponse <- EitherT.liftF(makeES8call(taxEnrolmentRequest))
+      result <- EitherT.fromEither(handleTaxEnrolmentServiceResponse(httpResponse)).leftFlatMap[Unit, Error] {
+                 error: Error =>
+                   logger
+                     .warn(
+                       s"Failed to allocate enrolments due to error: $error; will store enrolment details"
+                     )
+                   taxEnrolmentRepository
+                     .insert(taxEnrolmentRequest)
+                     .leftMap(error => Error(s"Could not store enrolment details: $error"))
                }
     } yield result
 
-  def handleTaxEnrolmentResponse(httpResponse: HttpResponse): Either[Error, Unit] =
+  def handleTaxEnrolmentServiceResponse(httpResponse: HttpResponse): Either[Error, Unit] =
     httpResponse.status match {
       case NO_CONTENT => Right(())
       case other      => Left(Error(s"Received error response from tax enrolment service with http status: $other"))
     }
 
   @SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements"))
```

Comment:
Is this `SuppressWarnings` necessary? It compiles on my machine without it

Reply:
no, not anymore - fixed by the new impl of this function